meta-mender-core/recipes-mender/mender-migrate-configuration/mender-migrate-configuration.bb

```diff
@@ -14,6 +14,17 @@ ALLOW_EMPTY_${PN} = "1"
 
 RDEPENDS_${PN} += "jq"
 
+# NOTE: Implicitly append split-mender-config (same as in mender.inc) and expect the
+# user to remove it explicitly in the local.conf
+PACKAGECONFIG[split-mender-config] = ",,,"
+PACKAGECONFIG_append = " split-mender-config"
+
+python do_check_config() {
+    if bb.utils.contains('PACKAGECONFIG', 'split-mender-config', True, False, d):
+        bb.fatal("'split-mender-config' needs to be removed for this migration recipe to work. Use PACKAGECONFIG_remove = \" split-mender-config\" in your local.conf")
```

Comment:
I think this isn't a good way. `PACKAGECONFIG` is supposed to be package specific, and using a global `PACKAGECONFIG_remove` is against convention. Tweaks are normally added in `.bbappend` files instead. Can we do this in a "softer" way, and in the state script detect if `mender.conf` is already present in `/data`? If it is, we can simply not update it.

Reply:
Done.